From 7bdf5b0eb3da6011e7a5b214a43a95b603e61030 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Sat, 9 Feb 2019 20:15:02 +0200
Subject: [PATCH 04/25] ucm-imx8m-mini: dts: Fix CSI reset GPIO

Fix the camera rst-gpio in accordance with SB-UCM-iMX8 Rev2.0 topology.

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts b/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts
index 991279f..a36af39 100644
--- a/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts
+++ b/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts
@@ -466,7 +466,7 @@
 		As of now the device is alwas out of reset & under the power
 		*/
 		pwn-gpios = <&pca9555 8 GPIO_ACTIVE_LOW>;
-		rst-gpios = <&pca9555 7 GPIO_ACTIVE_HIGH>;
+		rst-gpios = <&pca9555 3 GPIO_ACTIVE_HIGH>;
 
 		mclk = <24000000>;
 		mclk_source = <0>;
-- 
1.9.1

